### PR TITLE
Update PolicyFile Details section and add Policy Group Section

### PR DESCRIPTION
**Pull request by dishanktiwari2501**
**THE PR IS NOT READY FOR REVIEW**

### :nut_and_bolt: Description: What code changed, and why?

The Chef Infra Server page has been updated with the following sections:

- Policyfiles Details
- Cookbook Dependencies Button (under Policyfiles)
- Policy Group Section

### :chains: Related Resources

The reffered PRs are:

- https://github.com/chef/automate/pull/5398
- https://github.com/chef/automate/pull/5461
- https://github.com/chef/automate/pull/5484
- https://github.com/chef/automate/pull/5409
- https://github.com/chef/automate/pull/5431
- https://github.com/chef/automate/pull/5457
- https://github.com/chef/automate/pull/5485

### :white_check_mark: Checklist

**All PRs** must tick these:

- [x] I have read the [CONTRIBUTING document](https://github.com/chef/automate/blob/master/CONTRIBUTING.md).
- [x] All commits signed-off for the [Developer Certification of Origin](https://github.com/chef/chef/blob/master/CONTRIBUTING.md#developer-certification-of-origin-dco).

With occasional exceptions, all PRs **from Progress employees** must tick these:

- [x] Is the code clear? *(complicated code or lots of comments--subdivide and use well-named methods, meaningful variable names, etc.)*
- [x] Consistency checked? *(user notifications, user prompts, visual patterns, code patterns, variable names)*
- [x] Repeated code blocks eliminated? *(adapt and reuse existing components, blocks, functions, etc.)*
- [x] Spelling, grammar, typos checked? *(at a minimum use `make spell` in any component directory)*
- [x] Code well-formatted? *(indents, line breaks, etc. improve rather than hinder readability)*

All PRs **from Progress employees** should tick these if appropriate:

- [x] Tests added/updated? (all new code needs new tests)
- [x] Docs added/updated? (all customer-facing changes)

*Please add a note next to any checkbox above if you are NOT ticking it.*

### CODE REVIEW

File: components/docs-chef-io/content/automate/chef_infra_server.md [hunk 1]
```diff
@@ -467,16 +489,32 @@ Use the search bar on the Policyfile tab (**Chef Infra Server > Policyfile**) to
...
 
-Select a Policyfile to view the **METADATA**, **Content**, **Attributes**, and **Revision ID** of that Policyfile.
+Select a Policyfile to view the **METADATA**, **Content**, **Attributes**, **Revision ID**, and **Cookbook Dependencies** of that Policyfile.
 
 {{< figure src="/images/automate/details-of-policyfiles.png" alt="Details of Policyfiles">}}
 
-The **Content** tab contains the list of all the *included policies* and *run lists*, whereas the **Attributes** tab displays the *default* and the *override* attributes of a Policyfile.
+##### Content Tab
+
+The **Content** tab contains the list of all the *Included Policies* and *Run Lists*. Select a specific policyfile from the *Included Policies* to view the details.
```
Comment (tas50) [inline components/docs-chef-io/content/automate/chef_infra_server.md:498]:
```suggestion
The **Content** tab contains the list of all the *Included Policies* and *Run Lists*. Select a specific Policyfile from the *Included Policies* to view the details.
```

**Review by kagarmoe — APPROVED** [review]
tenor-5656693

**Review by himanshi-chhabra — APPROVED** [review]
LGTM :smiley: 
 Thanks @dishanktiwari2501

**Comment by kalroy** [comment]
@dishanktiwari2501 This PR needs to pass Netlify which is failing with the following error:
```
Start building sites …
1:28:20 PM: Total in 1393 ms
1:28:20 PM: Error: Error building site: "/opt/build/repo/components/docs-chef-io/chef-web-docs/_vendor/github.com/chef/automate/components/docs-chef-io/content/automate/chef_infra_server.md:166:112": failed to extract shortcode: template for shortcode "fontawesome" not found
1:28:20 PM: Makefile:10: recipe for target 'preview_netlify' failed
1:28:20 PM: make: *** [preview_netlify] Error 255
```

**Review by kagarmoe — APPROVED** [review]
Good to merge once it passes the build.

**Pull request merged**